apps/backend/src/subject-ratings/subject-ratings.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { PrismaService } from 'nestjs-prisma';
+
+import { CreateSubjectRatingDto } from './dto/create-subject-rating.dto';
+import { UpdateSubjectRatingDto } from './dto/update-subject-rating.dto';
+import { SubjectRating } from './entities/subject-rating.entity';
+
+@Injectable()
+export class SubjectRatingsService {
+  constructor(private prisma: PrismaService) {}
+
+  async create(data: CreateSubjectRatingDto): Promise<SubjectRating> {
+    return await this.prisma.subjectRating.create({ data });
```

Comment:
We need to worry about updating the subjects ratings as well here.

Reply:
Ofc now, that we dont have subjects yet, it is not that big of a problem, just something to keep in mind.